build/MUX-Release.yml

```diff
@@ -1,15 +1,8 @@
 name: $(BuildDefinitionName)_$(date:yyMM).$(date:dd)$(rev:rrr)
 variables:
   minimumExpectedTestsExecutedCount: 35  # Sanity check for minimum expected tests to be reported
+  Packaging.EnableSBOMSigning: true
 jobs:
-- job: ComponentDetection
```

Comment:
Is removing this related to SBOM?

Reply:
No, I'm just fixing it in passing. I meant to add a note about that.

Running ComponentDetection in its own Job is not the correct usage. It needs to run in the same Job as the build so that it can detect all the nuget packages that got restored. Running it in its own Job means that it doesn't really do anything.

This was not a huge deal, since this task gets automatically injected into the Pipeline in the correct place via policy anyway. But adding it explicitly seems better practice than relying on an auto-inject policy.

Reply:
Note that I moved ComponentGovernanceComponentDetection from here to MUX-BuildDevProject-Steps.yml.